src/help.ts

```diff
@@ -3,36 +3,36 @@ const help = `
 
     Generate Cordova resources for one or all platforms.
 
-    Without specifying options for source images, this tool will look for
-    'icon.(png|jpg)' and 'splash.(png|jpg)' files inside 'resources/'. To use
-    'res/' for the resources directory, specify '--resources res'.
+    By default, this tool will look for 'icon.(png|jpg)' and 'splash.(png|jpg)'
+    files inside 'resources/'. If an 'icon-foreground.(png|jpg)' file is found
```

Comment:
Maybe icon-background should also be mentioned here? Or what will happen if only an icon-foreground file is found?

Reply:
It will print an error message about it.